src/commands.rs

```diff
@@ -36,13 +38,80 @@ pub enum EnvAction {
     Remove,
 }
 
+/// Define what strategy to use when choosing privilege scalation programs.
+#[allow(dead_code)]
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum ElevationStrategy {
+    Auto,
+    Prefer(PathBuf),
+    Force(&'static str),
+}
```

Comment:
```suggestion
/// Strategy for choosing a privilege elevation program.
/// - `Auto`: try supported programs in fallback order.
/// - `Prefer(PathBuf)`: try the specified program, then fallback.
/// - `Force(&'static str)`: use only the specified program, error if not available.
```

In case myself or another contributor has trouble remembering what the enum is for.